src/utils/chunkUpload.ts

```diff
@@ -0,0 +1,25 @@
+import api from '../api'
+
+const CHUNK_SIZE = 100 * 1024 * 1024

```

Comment:
Please add a comment briefly describing why this is the chunk size

Reply:
Done